class UnauthorizedError(Exception):
    pass


class NotFoundException(Exception):
    pass
